src/api/test-controller.js

```diff
@@ -260,6 +260,10 @@ export default class TestController {
     _expect$ (actual) {
         return new Assertion(actual, this);
     }
+
+    _debugger$ () {
```

Comment:
It shouldn't be a function, but getter instead:
```js
await t
       .debugger
       .click(...)
        
```

See how it's done here for `t.ctx`: https://github.com/DevExpress/testcafe/blob/master/src/api/test-controller.js#L117 You need to rebase to the latest source.